eng/scripts/Language-Settings.ps1

```diff
@@ -5,6 +5,50 @@ $packagePattern = "*.zip"
 $MetadataUri = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/python-packages.csv"
 $BlobStorageUrl = "https://azuresdkdocs.blob.core.windows.net/%24web?restype=container&comp=list&prefix=python%2F&delimiter=%2F"
 
+function Get-AllPackageInfoFromRepo
```

Comment:
We should allow a directory to be passed so that we can scope this searching to make it more efficient/scoped as needed. And when you pull the python code into its own script you can just take in that scope path as a parameter and then you can use this script for both the get all or get one case here.